src/docs/GetStarted/Developers/npmScriptTutorial/index.mdx

```diff
@@ -137,11 +137,11 @@ then edit the file and add the folowing code :
 module.exports = require('@mozaic-ds/css-dev-tools/styleLintConfig.js')
 ```
 
-Run the `npm run css:lint-fix` command. Nothing should appen, because the code you copied as no error (let's hope).
+Run the `npm run css:lint-fix` command. Nothing should happen, because the code you copied as no error (let's hope).
```

Comment:
isn't it: 
because the code you copied **HAS** no error

❓

Reply:
LOL 
I missed this one :scream: